### PR TITLE
Add Harvest Moon 3D: The Tale of Two Towns

### PR DESCRIPTION
**Pull request by Spekular**
Adds Harvest Moon 3D: The Tale of Two Towns.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Spekular** [comment]
Missing alternate regions ATM as I'm unsure where to obtain their Title IDs. No github issues listed as there are none related to this game (based on a search for "harvest moon" in the Citra repo, only issues related to "A New Beginning" show up)

File: games/harvest-moon-3d-the-tale-of-two-towns/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Harvest Moon 3D: The Tale of Two Towns"
+description = "Harvest Moon: The Tale of Two Towns is a farming simulation video game developed by Marvelous Entertainment for the Nintendo DS as part of the Story of Seasons series. It was published by Marvelous Interactive in Japan, Natsume in North America, and Rising Star Games in Europe."
```
Comment (Leo626) [inline games/harvest-moon-3d-the-tale-of-two-towns/game.dat]: > for the Nintendo DS

should be

> for the Nintendo 3DS

**Comment by Leo626** [comment]
Looked up the title IDs. `000400000007A300` for `eur` and `00040000001CC400` for `jpn`.

**Comment by robzombie91** [comment]
http://www.3dsdb.com for future research spekular

**Comment by Spekular** [comment]
For the record: The Japanese release  has a "+" at the end ("Bokujou Monogatari: Futago no Mura +") and was released waaaaay later than the rest. (https://gamefaqs.gamespot.com/3ds/224090-bokujou-monogatari-futago-no-mura-/data)

Other than that I think (hope) I've fixed the issues pointed out.

**Pull request merged**